menshen/server.js

```diff
@@ -16,6 +16,8 @@ const SSHClient = require("ssh2").Client;
 
 async function main() {
 	io.on("connection", function (socket) {
+		console.log(`\r\n=== [SERVER] Connection to Client Established ===\r\n`);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/KingWaffleIII/kommander/pullRequest?prid=9134818)
```suggestion
		console.log("\r\n=== [SERVER] Connection to Client Established ===\r\n");
```